lib/KeyGenUtils.py

```diff
@@ -1,11 +1,91 @@
-import random
-import string
-import socket
+import uuid
+import getpass
+import Checksum
 
-def randomize(stringLength = 10):
-    lettersAndDigits = string.ascii_letters + string.digits
-    return ''.join(random.choice(lettersAndDigits) for i in range(stringLength))
+# Every Time Key will be different but gets verify only from the source it generated.
+# Key have the length of 108 characters ends with '=' symbol.
+# Please install pycryptodome before run this file. Use cmd 'pip install pycryptodome'.
+# Change the folder to Crypto from crypto.
 
 
-#def generateHash(systemIP, MAC):
```

Comment:
Write proper comments, comments are necessary

Reply:
This line is from your code, Sir.